bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/ghost/GhostCB.scala

```diff
@@ -49,12 +52,18 @@ final class GhostCB(obsComp: ISPObsComponent) extends AbstractObsComponentCB(obs
       // must operate on an SPSkyObject.
       def coordParam(so: SPSkyObject, name: Option[String],
                      raDeg: String, decDeg: String,
-                     raHMS: String, decDMS: String) : Unit = so.getCoordinates(None).foreach { c =>
-        name.foreach(n => config.putParameter(systemName, StringParameter.getInstance(n, so.getName)))
-        config.putParameter(systemName, StringParameter.getInstance(raDeg,  c.ra.formatDegrees))
-        config.putParameter(systemName, StringParameter.getInstance(decDeg, c.dec.formatDegrees))
-        config.putParameter(systemName, StringParameter.getInstance(raHMS,  c.ra.formatHMS))
-        config.putParameter(systemName, StringParameter.getInstance(decDMS, c.dec.formatDMS))
+                     raHMS: String, decDMS: String) : Unit = {
+        val coords = so match {
+          case c: SPCoordinates => c.getCoordinates.some
+          case t: SPTarget => t.getProperMotion.map(_.calculateAt(t.getTarget, Instant.now())).orElse(t.getCoordinates(None))
+        }
+        coords.foreach { c =>
+          name.foreach(n => config.putParameter(systemName, StringParameter.getInstance(n, so.getName)))
+          config.putParameter(systemName, StringParameter.getInstance(raDeg, c.ra.formatDegrees))
```

Comment:
From the UI it looks like this contains the degree symbol? Is that correct? That could cause troubles downstream. I'd expect this to be sent as a double rather than a string